testing/trino-testing/src/main/java/io/trino/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -108,6 +108,35 @@ public void testAggregation()
                 "SELECT regionkey, avg(CAST(nationkey AS double)) FROM nation GROUP BY regionkey");
     }
 
+    @Test
+    public void testTopN()
```

Comment:
This partially duplicates https://github.com/trinodb/trino/blob/69e693eb0015d5c356e14b7d4105dc315fc9793e/testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueries.java#L746-L809.
Let's keep things in one place.

See also https://github.com/trinodb/trino/pull/6935, but even without that PR, this applies.